services/spar/test-integration/Test/Spar/AppSpec.hs

```diff
@@ -10,14 +10,13 @@
 
 module Test.Spar.AppSpec (spec) where
 
-import Imports
 import Bilge
-import Control.Exception
 import Control.Lens
 import Data.Id
 import Data.String.Conversions
 import Data.UUID as UUID
 import Data.UUID.V4 as UUID
+import Imports
```

Comment:
I fixed this, but I am sloppy with imports because I think it's a waste of time to think about how they should be ordered.  Looking forward to brittany!